tests/handlers/test_device.py

```diff
@@ -522,6 +524,14 @@ def test_dehydrate_and_rehydrate_device(self) -> None:
 
         self.assertEqual(user_info.device_id, retrieved_device_id)
 
+        # make sure the user device has the refresh token
+        if _refresh_token:
```

Comment:
Why would there *not* be a refresh token if we requested one?

Please rename `_refresh_token` to `refresh_token` now that it is being used.

Reply:
it was intended to calm the linter down since `refresh_token` fn takes `refresh_token: str` and `register_device` returns `Optional[str]`.

Reply:
`assert refresh_token is not None` will solve that and check that it is true.